schematic/manifest/generator.py

```diff
@@ -1303,7 +1306,7 @@ def set_dataframe_by_url(
 
         return sh
 
-    def get_dataframe_by_url(self, manifest_url: str) -> pd.DataFrame:
+    def get_dataframe_by_url(self, manifest_url: str, strict=None) -> pd.DataFrame:
```

Comment:
I saw that `strict` is not being used. Could you remove it?

Reply:
Done!